standards/browser-support-table.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Browser Support Table</title>
+    <link rel="stylesheet" type="text/css" href="https://unpkg.com/capital-framework@5.0.3/dist/capital-framework.min.css">
+</head>
+<body>
+
+</body>
+</html>
+
+<table cellspacing="0" cellpadding="0" class="o-table o-table__striped o-table__stack-on-small o-table__entry-header-on-small">
```

Comment:
The content should go inside the `<body>` :)

Reply:
best code review ever

Reply:
totally meant to add this in a separate branch anyway. i'm develop